decentraland/redux/reducers.js

```diff
@@ -66,6 +67,8 @@ function myAssets(state = { list: [] }, action) {
   switch (type) {
     case ADD_TO_MY_ASSETS_LIST:
       return { ...state, list: [myAsset, ...state.list] };
+    case REMOVE_MY_ASSET_FROM_LIST:
+      return { ...state, list: state.list.filter(val => val !== myAsset) };
```

Comment:
I wonder if this would be more readable split out into a couple lines of code. Certainly a matter of personal preference and not important in a vacuum, but having the style match throughout the codebase has some value for understandability. Thoughts?

Reply:
Agree. I'll do that.

Reply:
Done.

Reply:
Thanks! Resolving